src/providers/config-provider.tsx

```diff
@@ -10,20 +14,57 @@ export const ConfigContext = createContext<IUIKitConfig>({
   globalTheme: {},
   excludes: [],
   getConfig: () => ({}),
+  getUiKitConfig: () => ({}),
 });
 
 export const ConfigProvider = ({ children, configs }: IConfigProviderProps) => {
-  const configFile = configs;
   const getConfig = (id: string) => {
-    const value = configFile.customizations[id];
+    const value = configs.customizations[id];
     return value;
   };
 
-  const globalTheme = configFile.theme;
-  const excludes = configFile.excludes;
+  const getCorrectIDs = ({
+    page,
+    component,
+    element,
+  }: IUIKitConfigOptions): string[] => {
+    if (!component && !element) return [`${page}/*/*`];
+    if (!element) return [`${page}/${component}/*`, `*/${component}/*`];
+    return [
+      `${page}/${component}/${element}`,
+      `${page}/*/${element}`,
+      `*/${component}/${element}`,
+      `*/*/${element}`,
+    ];
+  };
+
+  const getCorrectConfig = (
+    correctIds: string[]
+  ): Record<string, string | string[] | Record<string, string>> => {
+    for (let i = 0; i < correctIds.length; i++) {
+      const correctId = correctIds[i];
+      const correctConfig = configs.customizations[correctId];
+      if (correctConfig) return correctConfig;
+    }
```

Comment:
Can you use `Array.find` instead for this loop

Reply:
We want to break the loop as soon as we found the element. 
I am not sure array.find will break if we find the specific element
What do you think?

Reply:
tested and it worked so I updated with Array,.find Thanks xD